### PR TITLE
RPI lab 2

### PR DESCRIPTION
**Pull request by LarisaPobol**

### CODE REVIEW

File: lab2/index.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+<title>Responsive PSD Template</title>
+<meta name="keywords" content="">
+<meta name="description" content="">
+<link rel="stylesheet"  href="styles.css">
+<link href="https://fonts.googleapis.com/css?family=Montserrat" rel="stylesheet">
+<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+</head>
+<body>
+<header>	
+	<div class="Nav-for-computer">
```
Comment (LarisaPobol) [inline lab2/index.html]: nav

File: lab2/index.html [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+	<a href="#">PSD WEBSITES</a>
```
Comment (LarisaPobol) [inline lab2/index.html]: ul > li > a

File: lab2/index.html [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+<footer>
+<h3>BlazRobar.com</h3>
+<div class="Columns">
+	<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam vehicula ipsum non leo tristique eget facilisis lacus commodo. Curabitur vitae justo lorem, non ornare nibh. Pellentesque pretium, tellus ac ultrices faucibus, nibh mauris dapibus lorem, vitae viverra nisl elit sit amet libero. Nam eu blandit risus.</p>
+	<p>Phasellus imperdiet mattis nulla. Integer gravida imperdiet congue. Proin vitae pretium augue. Donec est sem, mattis et blandit ac, blandit at arcu. Donec tempus tincidunt suscipit. Suspendisse eu vulputate lacus.</p>
+	<p class="Only-for-computer">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam vehicula ipsum non leo tristique eget facilisis lacus commodo. Curabitur vitae justo lorem, non ornare nibh. Pellentesque pretium, tellus ac ultrices faucibus, nibh mauris dapibus lorem, vitae viverra nisl elit sit amet libero. Nam eu blandit risus.</p>
+	<p class="Only-for-computer">Phasellus imperdiet mattis nulla. Integer gravida imperdiet congue. Proin vitae pretium augue. Donec est sem, mattis et blandit ac, blandit at arcu. Donec tempus tincidunt suscipit. Suspendisse eu vulputate lacus.</p>
```
Comment (LarisaPobol) [inline lab2/index.html]: To read: Yandex BEM

File: index.html [hunk 4]
```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
```
Comment (aliaksandrkrasnou) [inline index.html:1]: Why do you have two files index.html? Which one's correct?

**Comment by aliaksandrkrasnou** [comment]
Approved, good 👍

**Pull request closed without merging**